XMonad/Util/Loggers.hs

```diff
@@ -182,6 +186,72 @@ logLayout = withWindowSet $ return . Just . ld
 logCurrent :: Logger
 logCurrent = withWindowSet $ return . Just . W.currentTag
 
+-- | Log the given string, as is.
+logConst :: String -> Logger
+logConst = return . Just
+
+-- | Like they 'maybe' combinator, but for  loggers. If the given logger
+-- gives @Nothing@, the default logger is returned.
+-- For example, to display a quote when no windows are on the screen,
+-- you can do:
+--
+-- > logMaybe logTitle (logConst "Hey, you, you're finally awake.")
+logMaybe :: Logger -> Logger -> Logger
+logMaybe l d = l >>= maybe d logConst
```

Comment:
I think, since you're explicitly mentioning `fromMaybe`, these arguments should be the same as that function, i.e. the default first and then the thing you're looking at

Reply:
Alternatively it could be named logFallback or logDefault or something and the order can stay.

Reply:
Not quite, a `fromMaybe` unwraps what's inside the `Maybe` with a fallback, we don't want it to be unwrapped 

I guess the name is confusing, I'll go with one of the suggestions of @liskin

Reply:
I have no qualms about the order as such, but when another function is references as essentially "this is like that" I think the order of arguments should be the same :) 

How it is right now is completely fine!